pkgs/by-name/he/hexxy/package.nix

```diff
@@ -0,0 +1,34 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  gitUpdater,
+}: let
+  pname = "hexxy";
+  version = "unstable-2024-06-07";
+in
```

Comment:
nit: not need to put `pname` and `version` in `let`.

Reply:
You're right, I've previously did something that'd require either `rec` or this `let in` block. Changing shortly.